ansible/playbook/tasks/create-lvm.yml

```diff
@@ -0,0 +1,64 @@
+- hosts: localhost
+  become: yes
+  gather_facts: yes
+  tasks:
+    - name: Stop supervisord
+      service:
+        name: supervisord
+        state: stopped
+
+    - name: "Copy /srv dir to safe location"
+      synchronize:
+        src: "/srv/"
+        dest: "/srv-copy"
+        recursive: yes
+
+    - name: Delete old /srv directory
+      file:
+        state: absent
+        path: /srv/
+
+    - name: Create Volume Group
+      lvg:
+        vg: DataVG
+        pvs: "/dev/nvme1n1"
+
+    - name: Create Thin Pool
+      register: thin_pool
+      failed_when: "thin_pool is failed and 'Sorry, no shrinking of DataLV to 0 permitted' not in thin_pool.msg"
+      lvol:
+        lv: DataLV
+        vg: DataVG
+        size: 100%FREE
+        opts: --thinpool ThinPool -V 50G
+
+    - name: Format LVM
+      filesystem:
+        fstype: xfs
+        dev: /dev/DataVG/DataLV
+        opts: -L DATA
+
+    - name: Mount
+      mount:
+        name: "/srv"
+        src: LABEL=DATA
+        fstype: xfs
+        opts: defaults,nofail
+        state: mounted
+
+    - name: Copy files to new partition
+      synchronize:
+        src: "/srv-copy/"
+        dest: "/srv/"
```

Comment:
```suggestion
        dest: "/srv"
```

Reply:
Otherwise we are inconsistent with https://github.com/percona/pmm-update/blob/main/ansible/playbook/tasks/roles/nginx/files/conf.d/pmm.conf#L187-L193

Reply:
It's just different different types of copying.
If we have a slash at the end, then we copy directory `/srv-copy` to `/srv`. Like `/srv/srv-copy/file1` or `/srv/srv-copy/file2`
If we don't have a shash at the end, then we copy files from /srv-copy to /srv/. Like: `/srv/file1` `/srv/file2`

Reply:
Yes, I know. I don't see how copying to /srv-copy (L187-193) is different from when you copy everything back.